internal/rules/resources.go

```diff
@@ -11,12 +11,12 @@ import (
 	"github.com/crusttech/crust/system/types"
 )
 
-type Access string
+type Access int
 
 const (
```

Comment:
Provide variables for these types so we can continue to use `rules.Allow` when we use the rules pkg